frontend/components/AddToCartBar/components/AddToCartButton/index.jsx

```diff
@@ -82,8 +84,11 @@ class AddToCartButton extends PureComponent {
         onClick={this.handleClick}
         data-test-id="addToCartBarButton"
         aria-hidden={!!itemCount}
+        type="button"
       >
-        <I18n.Text string={!itemCount ? 'product.add_to_cart' : 'product.go_to_cart'} />
+        <Portal name={RECHARGE_ADD_TO_CART_TEXT_PORTAL} props={itemCount}>
```

Comment:
I think the props should be given a value like 
```props={{ itemCount }}```
![image](https://user-images.githubusercontent.com/11764719/73306585-222cd800-41e2-11ea-828f-993574df425a.png)